tests/tpg300.py

```diff
@@ -52,7 +55,7 @@ def _set_pressure(self, expected_pressure, channel):
 
     def _set_units(self, unit):
         self._lewis.backdoor_run_function_on_device("backdoor_set_unit", [unit.value])
-        self._ioc.set_simulated_value("SIM:UNITS", unit.name)
+        self.ca.set_pv_value("UNITS:SP:RBV", unit.name)
```

Comment:
Writing to a setpoint readback doesn't usually make sense... a setpoint readback should be read directly from the hardware/emulator.

Can we set the `SIM` pv instead if this is only for recsim?